packages/sdk/src/types/Redemptions.ts

```diff
@@ -156,3 +152,65 @@ export type SimpleRollback = Pick<
 		points: number
 	}
 }
+
+export interface RedemptionsRedeemStackableParams {
+	options?: StackableOptionsRedemption
+	redeemables: StackableRedeemableParams[]
+	session?: ValidationSessionParams
+	order?: OrdersCreate
+	customer?: CustomersCreateBody
+	metadata?: Record<string, any>
+}
+
+export type RedemptionsRedeemStackableRedemptionResult = RedemptionsRedeemResponse & {
+	redemption: string
+}
+
+export type RedemptionsRedeemStackableOrderResponse = OrdersCreateResponse & {
+	redemptions?: Record<
+		string,
+		{
+			date: string
+			rollback_id?: string
+			rollback_date?: string
+			related_object_type: 'redemption'
+			related_object_id: string
+			stacked: string[]
```

Comment:
when looking at code, it doesn't seem to be a rule that we return it
I guess it's not if that was a normal/single redemption